tests/test_utils.py

```diff
@@ -50,9 +49,7 @@ def test_address_valid(value, name, email):
     '\u001f@example.com',
     '"@example.com',
     '\"@example.com',
-    '`@example.com',
```

Comment:
is this really a valid address?

Reply:
Sure, according to https://en.wikipedia.org/wiki/Email_address this is allowed, but in practice many MTAs restrict usage to much fewer special characters.

Reply:
ok